packages/diagrams/frontend/sirius-components-diagrams/src/index.ts

```diff
@@ -33,6 +33,7 @@ export type { GQLDiagram, GQLNodeLayoutData } from './graphql/subscription/diagr
 export type { GQLEdge } from './graphql/subscription/edgeFragment.types';
 export { GQLViewModifier } from './graphql/subscription/nodeFragment.types';
 export type { GQLNode, GQLNodeStyle } from './graphql/subscription/nodeFragment.types';
+export { useActions } from './renderer/actions/useActions';
```

Comment:
If you need to export this hook (and I'm not sure you do), the type of the action and the type of the hook `UseActionsValue` should be exported too

Reply:
I'm not sure too, but if someone needs it for any reason, it would have been already exported